{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-4619",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/exchange.xforce.ibmcloud.com\/vulnerabilities\/168862",
                    "name": "ibm-mq-cve20194619-info-disc (168862)",
                    "refsource": "XF",
                    "tags": []
                },
                {
                    "url": "https:\/\/www.ibm.com\/support\/pages\/node\/1135101",
                    "name": "https:\/\/www.ibm.com\/support\/pages\/node\/1135101",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "IBM MQ and IBM MQ Appliance 7.1, 7.5, 8.0, 9.0 LTS, 9.1 LTS, and 9.1 CD could allow a local attacker to obtain sensitive information by inclusion of sensitive data within trace. IBM X-Force ID: 168862."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2020-03-16T16:15Z",
    "lastModifiedDate": "2020-03-16T17:19Z"
}